snap/info.go

```diff
@@ -409,6 +409,12 @@ type SlotInfo struct {
 	Apps      map[string]*AppInfo
 }
 
+// SocketInfo provides information on application sockets.
+type SocketInfo struct {
+	ListenStream string
+	SocketMode   string
```

Comment:
This should probably be uint32, which is the underlying type of os.FileMode. But definitely an integer, even if a different type, and the content in the yaml file should be an integer too. Octal notation also works in yaml.